views/jobs/card.ejs

```diff
@@ -0,0 +1,38 @@
+<!-- The card where the job from a reqruiter is -->
+<div class="col">
+    <div class="card border-success mb-3 m-0 p-0" style="max-width: 75%">
+      <div class="card-header bg-transparent border-muted"></div>
+      <div class="card-body text-success">
+        <h4 class="card-title"><%= title %></h5>
+        <h5 class="card-subtitle mb-2 text-muted"><%= company %></h6>
+        <h6 class="card-subtitle mb-2 text-muted"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-geo-alt" viewBox="0 0 16 16">
+          <path d="M12.166 8.94c-.524 1.062-1.234 2.12-1.96 3.07A31.493 31.493 0 0 1 8 14.58a31.481 31.481 0 0 1-2.206-2.57c-.726-.95-1.436-2.008-1.96-3.07C3.304 7.867 3 6.862 3 6a5 5 0 0 1 10 0c0 .862-.305 1.867-.834 2.94zM8 16s6-5.686 6-10A6 6 0 0 0 2 6c0 4.314 6 10 6 10z"/>
+          <path d="M8 8a2 2 0 1 1 0-4 2 2 0 0 1 0 4zm0 1a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/>
+        </svg> <%= location %></h6>
+        <h7 class="card-subtitle mb-2 text-muted"><%= salary %></h6>
+        <p class="card-text">
+            <% let words = description.split(" ")%>
+              <% if (words.length > 40){ %>
+                <% let shortDesc = words.slice(0,40).join(" ") + "..."%>
+                <%= shortDesc %>
+              <% } else { %>
+              <%= description %> 
+              <% } %>
+           </p>
+
+        <a href="/jobs/<%= id %>" class="btn btn-outline-primary">Show more</a>
+        <br>
+        <a href="/jobs/<%= id %>/edit" class="btn text-muted float-left"><svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="bi bi-pencil" viewBox="0 0 16 16">
+          <path d="M12.146.146a.5.5 0 0 1 .708 0l3 3a.5.5 0 0 1 0 .708l-10 10a.5.5 0 0 1-.168.11l-5 2a.5.5 0 0 1-.65-.65l2-5a.5.5 0 0 1 .11-.168l10-10zM11.207 2.5L13.5 4.793 14.793 3.5 12.5 1.207 11.207 2.5zm1.586 3L10.5 3.207 4 9.707V10h.5a.5.5 0 0 1 .5.5v.5h.5a.5.5 0 0 1 .5.5v.5h.293l6.5-6.5zm-9.761 5.175l-.106.106-1.528 3.821 3.821-1.528.106-.106A.5.5 0 0 1 5 12.5V12h-.5a.5.5 0 0 1-.5-.5V11h-.5a.5.5 0 0 1-.468-.325z"/>
+        </svg> </a> 
+        <a href="/jobs/<%= id %>/delete" class="btn text-muted float-right"> <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="currentColor" class="bi bi-trash" viewBox="0 0 16 16">
+          <path d="M5.5 5.5A.5.5 0 0 1 6 6v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm2.5 0a.5.5 0 0 1 .5.5v6a.5.5 0 0 1-1 0V6a.5.5 0 0 1 .5-.5zm3 .5a.5.5 0 0 0-1 0v6a.5.5 0 0 0 1 0V6z"/>
+          <path fill-rule="evenodd" d="M14.5 3a1 1 0 0 1-1 1H13v9a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V4h-.5a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1H6a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1h3.5a1 1 0 0 1 1 1v1zM4.118 4L4 4.059V13a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1V4.059L11.882 4H4.118zM2.5 3V2h11v1h-11z"/>
+        </svg> </a> 
+
+    </div>
+      <div class="card-footer bg-transparent border-muted"></div>
```

Comment:
this div is empty, so as suggestion I would move the icons to this card-footer like this (padding-top and padding-bottom reduced to 0.5rem):
![grafik](https://user-images.githubusercontent.com/38247392/103536117-323d4500-4e92-11eb-807c-7be837630b2f.png)

Reply:
perfect i was trying to do this but didn't figure out how, thanks!